app/models/collection.rb

```diff
@@ -128,6 +129,7 @@ def self.available_collection_types
       'Project Groups',
       'Reports',
       'Cohorts',
+      'Supplemental Data Sets',
```

Comment:
Probably not worth doing right now, but looking at the number of places you had to touch in this one file, I think Collections could use a bit of yak shaving.  If we had a single constant hash that defined a bunch of bits, like:
```
{
 supplemental_data_sets: {
  title: 'Supplemental Data Sets',
  source_class: 'HmisSupplemental::DataSet',
  name_method: :name,

},
...
}
```
I think we could add new types of collections by just adding the variation to the constant.

Reply:
Yes, I was thinking the same. This could definitely use some TLC and the approach you have above could work well. I would do that in a different issue though; perhaps adding tests (if they don't exist already).
https://github.com/open-path/Green-River/issues/6801